[typings] shouldn't be logger instead of loggers?

Is it a typo [here](https://github.com/router5/router5/blob/c1c8d93a663f4b1fac8e5d9a39a3d000d03cf221/packages/router5/index.d.ts#L416)?
```typescript
declare module 'router5/plugins/loggers' {
```
maybe should be:
```typescript
declare module 'router5/plugins/logger' {
```

typescript compiler complains:
```text
Module not found: Can't resolve 'router5/plugins/loggers' in '...'
```

Changing the [L416](https://github.com/router5/router5/blob/c1c8d93a663f4b1fac8e5d9a39a3d000d03cf221/packages/router5/index.d.ts#L416) and [L31](https://github.com/router5/router5/blob/c1c8d93a663f4b1fac8e5d9a39a3d000d03cf221/packages/router5/index.d.ts#L31) it builds normal.

--- a/packages/router5/index.d.ts
+++ b/packages/router5/index.d.ts
@@ -28,7 +28,7 @@ declare module 'router5' {
         ActivationFn,
         ActivationFnFactory
     } from 'router5/core/route-lifecycle'
-    import loggerPlugin from 'router5/plugins/loggers'
+    import loggerPlugin from 'router5/plugins/logger'
     import transitionPath from 'router5-transition-path'
 
     type DoneFn = (err?: any, state?: State) => void
@@ -413,7 +413,7 @@ declare module 'router5/plugins/listeners' {
     export default listenersPluginFactory
 }
 
-declare module 'router5/plugins/loggers' {
+declare module 'router5/plugins/logger' {
     import { PluginFactory } from 'router5/core/plugins'
     const loggerPlugin: PluginFactory
     export default loggerPlugin

